src/options.tsx

```diff
@@ -11,8 +11,9 @@ import Table from "./Components/Table";
 import TooltipChar from "./Components/TooltipChar";
 import { noop } from "./consts";
 
+import i18n from "./i18n";
```

Comment:
`There should be at least one empty line between import groups - eslint(import/order)`
This line should be in the previous group